apps/devhub/templates/devhub/email/new-key-email.ltxt

```diff
@@ -0,0 +1,7 @@
+The following API Key was added to your account:
+{{ key }}
+
+If you believe this key was created in error, you can remove the key and
+disable access at the following location:
+
+<a href="{{ url('devhub.api_key') }}">{{ url('devhub.api_key') }}</a>
```

Comment:
I don't think you can mix and match html and text emails. You have to pick one. Thus, I think you should try this as a standalone URL, i.e. no anchor tag.